### ISSUE TITLE
BUG: wrong function formatting

### ISSUE THREAD

**Issue by rozhnev** [BaseMax/PHPAS#6]
When I try to format next code:
```<?php
function checkifhoused($productname, $supplier)
{
    $db = mysqli_connect(DB_SERVER, DB_USER, DB_PASS, DB_NAME);
    $sql = "SELECT * FROM " . TBL_PRODUCTS;
    $result = mysqli_query($db, $sql);
    while ($row = $result->fetch_assoc()) {
        if ($row['supplierid'] == $supplier and $row['name'] == $productname) {
            return true;
        }
    }
    return false;
}
```

the formater returns 
```<?php
functioncheckifhoused($productname, $supplier)
 {
	$db=mysqli_connect(DB_SERVER, DB_USER, DB_PASS, DB_NAME);
	$sql="SELECT * FROM ".TBL_PRODUCTS;
	$result=mysqli_query($db, $sql);
	while($row=$result->fetch_assoc()) {
		if($row['supplierid']==$supplierand$row['name']==$productname) {
			returntrue;
		}

	}

	returnfalse;
}
```

**Comment by BaseMax** [comment]
Hi,

Thanks for your report.

Which parts is wrong?
```
 {
```

or new line before `while`?

Oh...
Also `return` need a space after it!

**Comment by BaseMax** [comment]
If you are interested in correcting this, please do so. I appreciate. @rozhnev :+1:

**Comment by BaseMax** [comment]
Hi.
Is this issue still the case? Or has it been fixed?
Thanks for updating the topic. @rozhnev

**Comment by rozhnev** [comment]
Yep, the issue still actual. it is complicated, but I have not time now to care it right now.

**Comment by jasonc3107** [comment]
Hi,
I tried this and I am getting iscorrectly formatted code returned.

I use this

`<? include("presets.php"); include("includes/functions.php");
$_SESSION['currentPage'] = "rules.php";
define('pageTitle', 'rules');
$site_title = "Rules";

docStart($mysqli, $site_title, $metaDescription); ?>
<title><?php echo($site_title . ' - ' . titleApender); ?> - <?php echo towns; ?></title>
</head>

<body>
	<?php topMenuRES($mysqli, "rules"); ?>`

and get this return

`<?include("presets.php");
include("includes/functions.php");

$_SESSION['currentPage']="rules.php";

define('pageTitle', 'rules');

$site_title="Rules";


docStart($mysqli, $site_title, $metaDescription);
?>
<title><?phpecho($site_title.' - '.titleApender);
?>-<?phpechotowns;
?></title>
</head>

<body>
<?phptopMenuRES($mysqli, "rules");`

The php tag is joined to the next part of the file.  I used both versions of the php tag just to see if your script only works with one or the other, but it happends with both.

Also there is no space between the echo and the next part of the file.

Is this still being developed ?

Cheers
